### PR TITLE
Fix/replace gras references

### PR DESCRIPTION
**Pull request by bertearazvan**
- it fixes the references with DHI-GRAS repository, github namespace
- it fixes the modules in package json for both the core and the lab
- it proposes the `water-warriors-lab` package name
- temporarily suspends the pipe run for the lab

### CODE REVIEW

File: .github/workflows/main-lab.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ jobs:
         env:
           CI: true
           GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
-          NPM_TOKEN: ${{secrets.NPM_TOKEN}}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```
Comment (rihr) [inline .github/workflows/main-lab.yml:41]: Are these already added in DHI/react-components, or someone with owner permissions has to add them?

File: packages/react-components-lab/package.json [hunk 2]
```diff
@@ -2,21 +2,21 @@
   "version": "0.2.11",
   "license": "MIT",
   "main": "dist/index.js",
-  "name": "@dhi-gras/react-components-lab-test",
+  "name": "@dhi/water-warriors-lab",
```
Comment (rihr) [inline packages/react-components-lab/package.json]: I think I personally prefer the old one, matches the core package and is more neutral

File: .github/workflows/main-lab.yml (hunk 1 above)
Comment (bertearazvan) [reply .github/workflows/main-lab.yml:41]: The latter I believe. I think that if we are owners of that repo, we can use our personal OAuth

File: packages/react-components-lab/package.json (hunk 2 above)
Comment (bertearazvan) [reply packages/react-components-lab/package.json]: yes I agree, I have to change those

Comment (j08lue) [reply packages/react-components-lab/package.json]: I agree, let us rather stick to the conservative / boring naming for now. But let's keep the warrior vibe...

**Review by j08lue — CHANGES_REQUESTED** [review]

**Review by j08lue — APPROVED** [review]

**Comment by j08lue** [comment]
Ready for merging @bertearazvan 😄 If you merge here, perhaps I can merge the big PR on the upstream repo?

**Pull request merged**